app/models/email_suppression.rb

```diff
@@ -0,0 +1,8 @@
+class EmailSuppression < ApplicationRecord
+
```

Comment:
Can you please add a comment here saying that these are the names of Sendgrid suppression groups specified on Sendgrid? Otherwise it's kind of mysterious to someone who has no idea what's going on. Might also be worth noting that the IDs of the suppression groups that we actually use when sending emails are specified in `config/config.yml`.